.env

```diff
@@ -1,5 +1,6 @@
 REACT_APP_API_URI=
 REACT_APP_APPLICATION_NAME=$npm_package_name
+REACT_APP_ELIGIBLE_CITIES=helsinki,helsingfors
```

Comment:
should be array of string

Reply:
Can you please explain how that can be done? I don't think dotenv supports that.

Reply:
Actually true, i just checked, i think i put some config on webpack so it worked on Varaamo. How about we keep it, and we parse it into array when we use it?

Reply:
I think that is a perfect solution, marking conversation as resolved!